android/src/main/java/com/capacitorcommunity/videorecorder/VideoRecorderPlugin.java

```diff
@@ -535,6 +547,11 @@ class FrameConfig {
             borderRadius = object.getInteger("borderRadius", 0);
             JSObject ds = object.getJSObject("dropShadow");
             dropShadow = new DropShadow(ds != null ? ds : new JSObject());
+            if (object.has("mirror")) {
+                mirror = object.getBool("mirror");
+            } else {
+                mirror = true;
```

Comment:
Does this maintain the current behavior of the plugin as the default behavior?
I'll need to test this before merging.